subsys/bluetooth/mesh/rpl.c

```diff
@@ -0,0 +1,171 @@
+/*  Bluetooth Mesh */
+
+/*
+ * Copyright (c) 2017 Intel Corporation
+ * Copyright (c) 2020 Lingao Meng
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <string.h>
+#include <errno.h>
+#include <stdbool.h>
+#include <sys/atomic.h>
+#include <sys/util.h>
+#include <sys/byteorder.h>
+
+#include <net/buf.h>
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/mesh.h>
+
+#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_MESH_DEBUG_RPL)
+#define LOG_MODULE_NAME bt_mesh_rpl
+#include "common/log.h"
+
+#include "mesh.h"
+#include "adv.h"
+#include "net.h"
+#include "rpl.h"
+#include "settings.h"
+
+struct bt_mesh_rpl rpl_tabel[CONFIG_BT_MESH_CRPL];
```

Comment:
```suggestion
struct bt_mesh_rpl rpl_table[CONFIG_BT_MESH_CRPL];
```

Or just `rpl` or `replay_list`? replay_list_table is a bit redundant